lab-jessica/test/client-test.js

```diff
@@ -0,0 +1,28 @@
+const assert = require('assert');
+const Client = require('../lib/client.js');
+
+describe('Client Module', function() {
+  describe('#Client', function() {
+    it('creates new client', function() {
+      const clientExists = function() {
+        const client = new Client();
+        if(client) {
+          return true;
+        }
+      };
+      assert.strictEqual(true, clientExists());
+    });
+
+    it('client has properties socket, nickname, and id', function() {
+      const clientPropsExist = function() {
+        const client = new Client();
+        console.log(client);
+        if(client.hasOwnProperty('socket') && client.hasOwnProperty('nickname') && client.hasOwnProperty('id')) {
```

Comment:
Rather than doing a if conditional here, use mocha/chai methods to run your assertion/condition.